tests/_utils.py

```diff
@@ -28,11 +30,29 @@
 dt = lambda x: datetime.strptime(x, "%Y-%m-%d").date()  # noqa
 
 CFN_VALID_STATUS = ["CREATE_COMPLETE", "ROLLBACK_COMPLETE", "UPDATE_COMPLETE", "UPDATE_ROLLBACK_COMPLETE"]
+cloudwatch_client = boto3.client("cloudwatch")
 
 
 class ExecutionTimer:
-    def __init__(self, msg="elapsed time"):
+    def __init__(self, msg="elapsed time", test_name: Optional[str] = None):
```

Comment:
We will need to determine best way to _opt-in_ to these tests. For now, I was thinking we could set an Environment Variable in our codebuild jobs when running against `release-3.0.0` to ignore runs from unmerged PRs